resources/metadata/xsDeploy.yaml

```diff
@@ -97,6 +97,10 @@ spec:
         - STEPS
         secret: true
         mandatory: true
+        resourceRef:
+          - name: credentialsId
+            type: secret
+            param: username
```

Comment:
param is "user" here too. Might be where I copied it from to nexusUpload.

Reply:
see #1960